<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="Filters">
      <attribute name="title">
       <string>Filters</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Video</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QCheckBox" name="Filters_Video_signalstats">
                <property name="text">
                 <string>signalstats (video analysis statistics) </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Filters_Video_cropdetect">
                <property name="text">
                 <string>cropdetect (detect image boundaries)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Filters_Video_Psnr">
                <property name="text">
                 <string>PSNR (video field comparisons)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Filters_Video_Ssim">
                <property name="text">
                 <string>SSIM (video field comparisons)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Filters_Video_Idet">
                <property name="text">
                 <string>idet (interlacement detection)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Filters_Video_Palettegen">
                <property name="text">
                 <string>palettegen (generate palettes per frame)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Audio</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QCheckBox" name="Filters_Audio_astats">
                <property name="text">
                 <string>astats (time domain statistics about audio frames)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Filters_Audio_EbuR128">
                <property name="text">
                 <string>EBU R128 (perceived volume analysis)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Filters_Audio_aphasemeter">
                <property name="text">
                 <string>Audio Phase Meter (detect audio phase)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Tracks">
      <attribute name="title">
       <string>Tracks</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Video</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QRadioButton" name="Tracks_Video_First">
                <property name="text">
                 <string>First track only</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="Tracks_Video_All">
                <property name="text">
                 <string>All tracks</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Audio</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QRadioButton" name="Tracks_Audio_First">
                <property name="text">
                 <string>First track only</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="Tracks_Audio_All">
                <property name="text">
                 <string>All tracks</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Export (Not yet implemented)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QRadioButton" name="XmlGz_Prompt">
             <property name="text">
              <string>Prompt</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="XmlGz_Sidecar">
             <property name="text">
              <string>Automaticly save to .qctools.xml.gz sidecar file</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="XmlGz_Directory">
             <property name="text">
              <string>Automatically save to a specific directory:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="XmlGz_Directory_Name">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>~/.qctools/metadata</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Signalserver">
      <attribute name="title">
       <string>Signalserver</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QCheckBox" name="signalServerEnableAutoUpload_checkBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Upload to signalserver automatically</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>221</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QLineEdit" name="signalServerUrl_lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>Signalserver URL (http://example.com)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QLineEdit" name="signalServerPassword_lineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QLineEdit" name="signalServerLogin_lineEdit">
         <property name="placeholderText">
          <string>User name</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QPushButton" name="testConnection_pushButton">
         <property name="text">
          <string>Test connection</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>220</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="0" colspan="4">
        <widget class="QLabel" name="connectionTest_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="signalServerEnable_checkBox">
         <property name="text">
          <string>Enable signalserver</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Warning: passwords are stored insecurely and transmit in plain text!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="4">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>Filters_Video_signalstats</tabstop>
  <tabstop>Filters_Video_cropdetect</tabstop>
  <tabstop>Filters_Video_Psnr</tabstop>
  <tabstop>Filters_Video_Ssim</tabstop>
  <tabstop>Filters_Video_Idet</tabstop>
  <tabstop>Filters_Audio_astats</tabstop>
  <tabstop>Filters_Audio_EbuR128</tabstop>
  <tabstop>Filters_Audio_aphasemeter</tabstop>
  <tabstop>Tracks_Video_First</tabstop>
  <tabstop>Tracks_Video_All</tabstop>
  <tabstop>Tracks_Audio_First</tabstop>
  <tabstop>Tracks_Audio_All</tabstop>
  <tabstop>XmlGz_Prompt</tabstop>
  <tabstop>XmlGz_Sidecar</tabstop>
  <tabstop>XmlGz_Directory</tabstop>
  <tabstop>XmlGz_Directory_Name</tabstop>
  <tabstop>signalServerUrl_lineEdit</tabstop>
  <tabstop>signalServerLogin_lineEdit</tabstop>
  <tabstop>signalServerPassword_lineEdit</tabstop>
  <tabstop>signalServerEnable_checkBox</tabstop>
  <tabstop>signalServerEnableAutoUpload_checkBox</tabstop>
  <tabstop>testConnection_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>signalServerEnable_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>signalServerEnableAutoUpload_checkBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
